### DISCUSSION TITLE
Feature request - Admin search and bulk action

### DISCUSSION THREAD

**Discussion by allanice001** [microfeed/microfeed#113] [Ideas]
I am very happy with running microfeed on https://dstif.io, and have been using it as a feed aggregator.

Everyso often, a scenario comes up where due to some automation i've built, i get duplicate entries, and would like to be able to bulk delete these duplications

having the ability to search within the admin interface would be a nice to have (search by title?)
Additionally having the ability to bulk select would also have a nice to have (manually deleting 100 entries iis not fun)

**Comment by kenara** [comment]
@allanice001, improvement to the admin UI, possibly including bulk deleting, is one thing and I'm sure @wenbinf is considering that and welcomes any pull requests.  

But I believe that your need to avoid duplicate entries should be the responsibility of your automation, especially considering that it is probably not a requirement of many Microfeed users. 

Searching the items is also a highly individualised requirement and probably out of scope for Microfeed. But if your automation can add items to Microfeed and you don't have a huge number of items/operations, you might consider something like a free account with Algolia (https://dashboard.algolia.com/users/sign_up). Their free tier includes 10,000 search requests/mo and up to 1 Million records. Even knowing nothing about your automation, it's safe to say they have a library that you can integrate.

With Search configured to meet your requirements, you should be able to avoid adding duplicates in the first place. If, in the course of grooming your data you find duplicates, you can simply use the Microfeed API `DELETE` endpoint (https://www.microfeed.org/json/openapi#tag/Admin-API/operation/apiDeleteItemById) to remove them.
